mkdocs.yml

```diff
@@ -230,6 +264,7 @@ extra_css:
 - assets/stylesheets/branding.css
 - assets/stylesheets/edgey.css
 - assets/stylesheets/newexample.css
+- assets/stylesheets/extra.css
 extra_javascript:
   - assets/javascripts/version-select.js
```

Comment:
Yo will need to remove this to get rid of the old version selector dropdown

Reply:
per meeting with @ernestojeda today - removed